legend-engine-protocol-pure/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/packageableElement/mapping/PropertyPointer.java

```diff
@@ -14,13 +14,40 @@
 
 package org.finos.legend.engine.protocol.pure.v1.model.packageableElement.mapping;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.finos.legend.engine.protocol.pure.v1.PureProtocolObjectMapperFactory;
 import org.finos.legend.engine.protocol.pure.v1.model.SourceInformation;
 
+import java.io.IOException;
+
+@JsonDeserialize(using = PropertyPointer.PropertyPointerDeserializer.class)
 public class PropertyPointer
 {
-    @JsonProperty(value = "class")
-    public String _class;
+    public String owner;
     public String property;
     public SourceInformation sourceInformation;
+
+    public static class PropertyPointerDeserializer extends JsonDeserializer<PropertyPointer>
```

Comment:
Do we have a test for the old protocol? I want to ensure even if we send in the old protocol, your custom deserializer works

Reply:
I have added a test where we assert on building new protocol from the json with fields from old protocol. Would that be sufficient?

Reply:
yep. Thanks!